ansible/roles/common/tasks/configs.yml

```diff
@@ -30,7 +30,15 @@
     owner: pi
     group: pi
     mode: 0644
-
+    
+- name: MeteorDemod config settings.ini file
```

Comment:
This failed for me on the first run, with the following error:
```
TASK [common : MeteorDemod config settings.ini file] *********************************************************************
fatal: [localhost]: FAILED! => {"changed": false, "checksum": "b444a9fbcaeebec7550c33a7c619746adef63df7", "msg": "Destination /home/pi/.config/meteordemod not writable"}
        to retry, use: --limit @/home/pi/raspberry-noaa-v2/ansible/site.retry

PLAY RECAP ***********************************************
localhost                  : ok=5    changed=3    unreachable=0    failed=1 
```

I *suspect* what has happened is your deb has added files here (presumably hard coded to the pi user? which is a really bad idea given #461) and therefore they have been created owned by root.

Reply:
As I said, I'm not familiar eith Ansible and as a result, you will have to do that on your own. I just created some starting point which I think *could* work.

I used a deb file from MeteorDemod Github, it only works for the user named pi, he still haven't created deb file which coučd work for any user.